distributed-testing/install-bzt-jmeter.yml

```diff
@@ -15,8 +15,9 @@ settings:
   artifacts-dir: ./tmp
 
 modules:
+  # https://gettaurus.org/docs/JMeter/
   jmeter:
-    properties:
-      hostname: test
+    force-ctg: true # required for higher concurrency thread groups
+    detect-plugins: true
```

Comment:
What are the plugins that are being detected?

Reply:
Jmeter Plugins. When true, it's supposed to automatically detect what plugins to install